README.org

```diff
@@ -48,6 +48,16 @@ Lispyville can also be used without lispy:
 ;; ...
 #+end_src
 
+If you're using `use-package` and `general`. The hooks and key-themes in the
+following example should be configured:
+
+#+begin_src
+(use-package lispyville
+  :config
+  (general-add-hook '(emacs-lisp-mode-hook lisp-mode-hook) #'lispyville-mode)
```

Comment:
Put this in `:init` or use `:ghook` (e.g. `:ghook 'emacs-lisp-mode-hook 'lisp-mode-hook`).